packages/babel-plugin-transform-object-assign/README.md

```diff
@@ -1,6 +1,6 @@
 # babel-plugin-transform-object-assign
 
-Replace `Object.assign` with an inline helper.
+Replace `Object.assign` with an inline helper. If you are authoring an application, rather than a library, it is recommended that you use the `Object.assign` polyfill instead.
```

Comment:
Small nit, but we've been standardizing on adding a `>` before the descriptions.

See: https://github.com/babel/babel/blob/master/packages/babel-preset-es2017/README.md